File: gestionUI/src/app/shared/ticket.ts
export class Ticket{
    id : string;
    key : string;
    summary : string;
    description : string;
    type : string;

    constructor(id: string, key: string, summary : string , description: string , type : string) {
        this.id = id;
        this.key = key;
        this.summary = summary;
        this.description = description;
        this.type = type;
    }
}

File: gestion/src/main/java/com/hps/gestion/dto/Ticket.java
package com.hps.gestion.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Ticket {
    private String id;
    private String key;
    private String summary;
    private String description;
    private String type;
}
